src/PHPDocsMD/Console/PHPDocsMDCommand.php

```diff
@@ -20,19 +20,37 @@ class PHPDocsMDCommand extends \Symfony\Component\Console\Command\Command {
     const ARG_CLASS = 'class';
     const OPT_BOOTSTRAP = 'bootstrap';
     const OPT_IGNORE = 'ignore';
+    const OPT_VISIBILITY = 'visibility';
+    const OPT_METHOD_REGEX = 'methodRegex';
 
     /**
      * @var array
      */
     private $memory = [];
 
+    /**
+     * @var array
+     */
+    private $visibilityFilter = [];
+
+    /**
+     * @var string
+     */
+    private $methodRegex = '';
+
     /**
      * @param $name
      * @return \PHPDocsMD\ClassEntity
      */
     private function getClassEntity($name) {
         if( !isset($this->memory[$name]) ) {
             $reflector = new Reflector($name);
+	        if ( ! empty( $this->visibilityFilter ) ) {
```

Comment:
This more a WordPress code style and different to this package. I mean it should use PSR style, without whitespace.

Reply:
This is a topic for more files in this PR.

Reply:
Thanks for the feedback, I've updated it my latest commit.
Guilty as charges: working on WordPress most of my time my IDE code style settings are set to the WordPress one.

Reply:
Yes, I'm have also often this topic. In my repositories I leave a code-style config fiel to set the IDE, especially PhpStorm to the right style, like WP or PSR.